CRISPResso2/CRISPRessoCORE.py

```diff
@@ -2774,8 +2774,8 @@ def calculate_99_max(d):
         with open(mapping_stats_filename, 'w+') as outfile:
             outfile.write('READS IN INPUTS\tREADS AFTER PREPROCESSING\tREADS ALIGNED\tN_COMPUTED_ALN\tN_CACHED_ALN\tN_COMPUTED_NOTALN\tN_CACHED_NOTALN\n')
             outfile.write("\t".join([str(x) for x in[N_READS_INPUT, N_READS_AFTER_PREPROCESSING, N_TOTAL, aln_stats['N_COMPUTED_ALN'], aln_stats['N_CACHED_ALN'], aln_stats['N_COMPUTED_NOTALN'], aln_stats['N_CACHED_NOTALN']]]) + "\n")
-        crispresso2_info['aln_stats'] = aln_stats
-        crispresso2_info['mapping_stats_filename'] = os.path.basename(mapping_stats_filename)
+        crispresso2_info['running_info']['aln_stats'] = aln_stats
```

Comment:
This may cause confusion -- 'aln_stats' vs 'alignment_stats' -- perhaps break this out into running_info/alignment_stats/n_reads_computed_aligned etc?

Reply:
I agree, I have fixed this.